data/mods/Magiclysm/Spells/attunements/Gaias_Chosen.json

```
@@ -0,0 +1,35 @@
+[
+  {
+    "id": "gaiaschosen_samsara",
+    "type": "SPELL",
+    "spell_class": "GAIAS_CHOSEN",
+    "name": "Samsara",
+    "description": "You let your mana spin in the world's life flow, renewing your body as if prematurely reincarnated.",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 250000,
+    "final_casting_time": 50000,
+    "casting_time_increment": -6250,
```

the casting time for this is too high. i don't see a reason to have the decrement, so i'd probably do it this way:
```suggestion
    "base_casting_time": 60000,
```